src/lib/run-dev.js

```diff
@@ -348,7 +351,10 @@ async function invokeAction ({ actionRequestContext, logger }) {
   let actionFunction
   // catch errors when loading the action function
   try {
-    actionFunction = require(action.function)?.main
+    const actionFolder = path.join(distFolder, packageName, actionName)
+    // TODO: find a better way to get the actual folder name without this magic string
+    const actionPath = `${actionFolder}-temp`
```

Comment:
to not use the magic string, we may have to call the lib to build the actions directly, and not use `aio app build`

Reply:
We can't do this, since aio-lib-runtime uses [this magic string](https://github.com/adobe/aio-lib-runtime/blob/da90917ef69a02b0131d3abafa123f1b503948ae/src/build-actions.js#L173) and does not expose it